spec/wasabi/parser/no_message_parts_spec.rb

```diff
@@ -15,7 +15,7 @@
 
     it 'falls back to using the message type in the port element' do
       # Operation's input has no part element in the message, so using the message type.
-      expect(subject.operations[:save][:input]).to eq('Save')
+      expect(subject.operations[:save][:input]).to eq("SaveSoapIn")
```

Comment:
so if i understand correctly, this value is coming from here? https://github.com/savonrb/wasabi/blob/master/spec/fixtures/no_message_parts.wsdl#L51

How did this work before? it would fall back to here? https://github.com/savonrb/wasabi/pull/114/files#diff-42bb513f8ac3b44919b550f9374bdadf8ebc8130dd0f1776571d6aba51e07c2dR292